src/main/java/core/basesyntax/service/Parser.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.Instruction;
+import java.util.List;
+
+public interface Parser {
+    List<Instruction> parse(List<String[]> lines);
```

Comment:
why did you receive a list of string arrays here? seems to be an inconsistent approach.